ds3-autogen-c/src/main/resources/templates/TypeEnumConstant.tmplt

```diff
@@ -0,0 +1,7 @@
+<#include "CopyrightHeader.tmplt"/>
+
+typedef enum {
+<#list enumConstants as enum>
+    ${enum.getName()},
```

Comment:
very nice, thanks.  Some languages just ignore the trailing comma, so I wasn't sure if this would be  an issue and didn't worry about it at the time, but this is cleaner.